week_1/String/find_all_anagrams/find_all_anagrams.test.js

```diff
@@ -0,0 +1,16 @@
+const findAllAnagrams = require("./find_all_anagrams");
+
+describe("find_all_anagrams", () => {
+	test("return an empty array if short > long", () => {
+		expect(findAllAnagrams("oh", "fine")).toEqual([]);
+	});
+
+	test("return starting index if they contain the same characters", () => {
+		expect(findAllAnagrams("job", "boj")).toEqual([0]);
+	});
+
+	test("find all anagrams of a short string in a longer string", () => {
```

Comment:
nit: it's nice to see "happy path" tests first, i.e moving the last test here to be the first test